spec/spec_helper_acceptance.rb

```diff
@@ -3,7 +3,7 @@
 require 'voxpupuli/acceptance/spec_helper_acceptance'
 
 configure_beaker do |host|
-  install_module_from_forge('puppet-epel', '>= 3.0.1 < 4.0.0') if fact_on(host, 'os.name') == 'CentOS'
-  install_module_from_forge('puppetlabs-apache', '>= 5.7.0 < 7.0.0')
-  install_module_from_forge('puppetlabs-java', '>= 6.3.0 < 8.0.0')
+  install_puppet_module_via_pmt_on(host, 'puppet-epel', '>= 3.0.1 < 4.0.0') if fact_on(host, 'os.name') == 'CentOS'
```

Comment:
OracleLinux is failing because it's not installing the EPEL module here. That was also an issue previously, so it's not a regression. Just wondering how this ever worked.

Reply:
there's a high chance that when the tests passed the last time, acceptance tests weren't enabled for OracleLinux?

Reply:
Oh yes, that's very likely.